crv/crvShape.cc

```diff
@@ -535,7 +521,10 @@ static int markEdgesToFix(Adapt* a, int flag)
 
   ma::Mesh* m = a->mesh;
   ma::Entity* e;
-  ma::Entity* edges[3];
+
+  // markEdges could have upto 6 edges marked!!!
+  /* ma::Entity* edges[3]; */
```

Comment:
I would like to guess that maybe at some point in the future. Also, I think (need to do a bit more thinking on this) it would be better to deal with non-simplex cases separately. The current curving and curve adapt features rely heavily on the assumptions that we are dealing with simplexes.